tests/test_hba.py

```diff
@@ -378,6 +378,27 @@ def test_as_dict():
     }
 
 
+def test_hbarecord_constructor():
+    from pgtoolkit.hba import HBARecord
+
+    HBARecord(
+        conntype="local",
+        database="all",
+        user="all",
+        method="trust",
+        comment="This is comment!",
+    )
+    HBARecord(
+        **dict(
+            conntype="local",
+            database="all",
+            user="all",
+            method="trust",
+            comment="This is comment!",
+        )
+    )
```

Comment:
No `assert`? In fact, what's the point of this test?

Reply:
It indeed is useless. I though this was failing before the change but it was not. This was already valid, only Mypy was complaining. Removed.